source/authentication.rst

```diff
@@ -67,242 +71,18 @@ An authentication session uses the following sequence:
-``libsasl2`` is hard-coded to look in certain directories at build time.
-This location may be different depending on the installation method.
-
-In the configuration file, specify the following:
-
-.. code-block:: text
-
-   pwcheck_method: saslauthd
-   saslauthd_path: /var/run/saslauthd/mux
-   log_level: 5
-   mech_list: plain
-
-The first two entries (``pwcheck_method`` and ``saslauthd_path``)
-are required for ``mongod`` to successfully use the ``saslauthd`` service.
-The ``log_level`` is optional but may help determine configuration errors.
-
-.. seealso::
-
-   `SASL documentation: <https://www.cyrusimap.org/sasl/index.html>`_
-  
-Configuring ``mongod`` Server
------------------------------
-
-To enable external authentication, you must create a user with the **root** privileges in the ``admin`` database. If you have already created this user, skip this step. Otherwise, run the following command to create the admin user:
-
-.. code-block:: text
-
-   > use admin
-   switched to db admin
-   > db.createUser({"user": "admin", "pwd": "$3cr3tP4ssw0rd", "roles": ["root"]})
-   Successfully added user: { "user" : "admin", "roles" : [ "root" ] }
-
-Edit the :file:`etc/mongod.conf` configuration file to enable the external authentication:
-
-.. code-block:: yaml
-
-   security:
-     authorization: enabled
-
-   setParameter:
-     authenticationMechanisms: PLAIN,SCRAM-SHA-1
-
-Restart the ``mongod`` service:
-
-.. code-block:: bash
-
-   $ sudo systemctl restart mongod
-
-When everything is configured properly, you can use the :ref:`commands`.
-
-.. _commands:
-
-External Authentication Commands
-================================
-
-Use the following command to add an external user to the ``mongod`` server:
-
-.. code-block:: text
-
-   > db.getSiblingDB("$external").createUser( {user : "christian", roles: [ {role: "read", db: "test"} ]} );
-
-The previous example assumes that you have set up the server-wide
-admin user/role and have successfully authenticated as that user locally.
-
-.. note:: External users cannot have roles assigned in the admin database.
-
-When running the ``mongo`` client, a user can authenticate
-against a given database using the following command:
-
-.. code-block:: text
-
-   > db.getSiblingDB("$external").auth({ mechanism:"PLAIN", user:"christian", pwd:"secret", digestPassword:false})
-
-
-.. admonition:: Based on the material from **Percona Database Performance Blog**
-		
-   This section is based on the blog post *Percona Server for MongoDB Authentication Using Active Directory* by *Doug Duncan*:
-      https://www.percona.com/blog/2018/12/21/percona-server-for-mongodb-authentication-using-active-directory/
-  
+For configuration instructions, refer to :ref:`sasl`.
 
 .. _ldap-authorization:
 
 Authentication and Authorization with Direct Binding to LDAP
 ============================================================
-  
+
 This feature has been supported in MongoDB Enterprise since its version 3.4.
```

Comment:
Is it possible also to add version when this feature appeared in PSMDB?

Reply:
Added